src/main/resources/messages.properties

```
@@ -97,9 +97,9 @@ modal.whitelist.step.minecraft.label=Minecraft Name
 modal.whitelist.step.minecraft.invalid=Du musst einen g�ltigen Minecraft-Java Namen angeben.
 modal.whitelist.step.minecraft.already-whitelisted=Du bist bereits auf der Whitelist.
 modal.whitelist.step.minecraft.open=> Minecraft Name: `{0}`\n\n\
-Vielen Dank f�r deine Whitelist Anfrage. \
-Du wirst nun auf die Whitelist hinzugef�gt, sobald jemand aus dem Team Zeit findet.\
- Wir bitten um etwas Geduld.
+Vielen Dank f�r deine Whitelist Anfrage.\n\
+Du wirst nun auf die Whitelist hinzugef�gt, sobald jemand aus dem Team Zeit findet.\n\
+Wir bitten um etwas Geduld.
```

Maybe don't make the message ugly with three new lines?

```suggestion
Vielen Dank f�r deine Whitelist Anfrage. \
Du wirst nun auf die Whitelist hinzugef�gt, sobald jemand aus dem Team Zeit findet. \
Wir bitten um etwas Geduld.
```